### DISCUSSION TITLE
Hot reload events

### DISCUSSION THREAD

**Discussion by CrosRoad95** [dotnet/runtime#71216] [General]
Idea is simple, make possible to call specific action after hot reload reload the code.
Currently you can't hot reload things like: fluent validation, auto mapper and much more libraries that have fixed configuration - with some events, when programmer change mapping, validation ect then it can be detected and reloaded.

**Comment by HaloFour** [comment]
This sounds more like a [runtime](https://github.com/dotnet/runtime) ask?

**Reply by 333fred** [reply]
Transferred.

**Comment by jkotas** [comment]
Take a look at https://docs.microsoft.com/en-us/dotnet/api/system.reflection.metadata.metadataupdatehandlerattribute. It allows you to specify callbacks to call after the hot reload update.
